partials/vue.js

```diff
@@ -0,0 +1,214 @@
+'use strict';
+const vueRulesBase = require('eslint-plugin-vue/lib/configs/base').rules,
+      vueRulesEssentials = require('eslint-plugin-vue/lib/configs/essential').rules,
+      vue3RulesEssentials = require('eslint-plugin-vue/lib/configs/vue3-essential').rules,
+      vueRulesStronglyRecommended = require('eslint-plugin-vue/lib/configs/strongly-recommended').rules,
+      vue3RulesStronglyRecommended = require('eslint-plugin-vue/lib/configs/vue3-strongly-recommended').rules,
+      vueRulesRecommended = require('eslint-plugin-vue/lib/configs/recommended').rules,
+      vue3RulesRecommended = require('eslint-plugin-vue/lib/configs/vue3-recommended').rules;
+
+const silvermineRules = {
```

Comment:
It looks like this object still contains Vue-3-specific rules, which means they are getting mixed into the Vue 2 config. For example, `vue/no-deprecated-v-is` applies to Vue 3 only.

We probably need separate object structures for base rules, Vue 2 rules, and Vue 3 rules.